src/commands/aksAccount/aksAccount.ts

```diff
@@ -0,0 +1,71 @@
+import { QuickPickItem, Uri, env, window } from "vscode";
+import { failed } from "../utils/errorable";
+import { SubscriptionFilter, getFilteredSubscriptions, setFilteredSubscriptions } from "../utils/config";
+import { signIn } from "../../auth/azureAuth";
+import { SelectionType, getSubscriptions, getTenantIds } from "../utils/subscriptions";
+
+export async function signInToAzure(): Promise<void> {
+    await signIn();
+}
+
+type SubscriptionQuickPickItem = QuickPickItem & { subscription: SubscriptionFilter };
+
+export async function selectSubscriptions(): Promise<void> {
+    const allSubscriptions = await getSubscriptions(SelectionType.All);
+    if (failed(allSubscriptions)) {
+        window.showErrorMessage(allSubscriptions.error);
+        return;
+    }
+
+    if (allSubscriptions.result.length === 0) {
+        const noSubscriptionsFound =
+            "No subscriptions were found. Setup your account if you have yet to do so or check out our troubleshooting page for common solutions to this problem.";
+        const setupAccount = "Setup Account";
+        const openTroubleshooting = "Open Troubleshooting";
+        const response = await window.showInformationMessage(noSubscriptionsFound, setupAccount, openTroubleshooting);
+        if (response === setupAccount) {
+            env.openExternal(Uri.parse("https://aka.ms/AAeyf8k"));
```

Comment:
💡 I wonder what these alias are and worth to comment code the resolved name endpoint here.

Reply:
Great catch! That resolves to https://github.com/microsoft/vscode-azure-account/wiki/Troubleshooting#setup-your-azure-account which is specifically for the Azure Account extension that we're removing.

There is no general-purpose troubleshooting page for authentication in our extension, so I'm removing that option. For setting up an account I'm changing the URL to "https://azure.microsoft.com/" (which is what the Azure Account documentation links to).